ambry-api/src/main/java/com.github.ambry/config/HelixAccountServiceConfig.java

```diff
@@ -98,6 +101,15 @@
   @Default("false")
   public final boolean enableServeFromBackup;
 
+  /**
+   * Total number of previous versions of account metadata to keep in the system. Every update account http request would
+   * generate a new version. And when the number of versions surpasses this number, HelixAccountService will purge the
+   * oldest one to make room for the new one.
+   */
+  @Config(TOTAL_NUMBER_OF_VERSION_TO_KEEP)
+  @Default("100")
+  public final int totalNumberOfVersionToKeep;
```

Comment:
Ah, probably if this is set to 100, the deleteBlob is likely to not be an issue.